katas/content/teleportation/entangled_pair/solution.md

```diff
@@ -0,0 +1,8 @@
+Begin by understanding what is the output we are expected to give. The resultant state is a superposition of two states, $|00\rangle$ and $|11\rangle$. Starting with the state $|00\rangle$, which is a basis state of two qubit system, need is to introduce superposition in such a way that we get a total of two states. This can be achieved by applying Hadamard gate to any one of them, let's say qAlice qubit. 
```

Comment:
I would simplify this, similar to how superdense coding kata does it - mention that it's a well-known state that we've handled in "Multi-Qubit Gates" and "Preparing Quantum States" katas, and briefly list the steps and the states after that as a numbered list, but we don't need to justify the choice of steps

Reply:
Changed